osrf/vrx
# Tweak ball shooter mesh to make rviz happy

When launching rviz, its COLLADA parser was complaining about some empty elements. Here's an example:

```
[ERROR] [1634936987.984769294, 382.378000000]: Could not load resource [package://vrx_gazebo/models/ball_shooter/meshes/ball_shooter_launcher.dae]: Collada: package://vrx_gazebo/models/ball_shooter/meshes/ball_shooter_launcher.dae - Expected end of <keywords> element.
```

How to test it:

```
roslaunch vrx_gazebo vrx.launch extra_gazebo_args:="--verbose"
```

```
roslaunch wamv_gazebo localization_example.launch
```

```
roslaunch wamv_gazebo rviz_vrx.launch
```

You shouldn't see any errors in the rviz terminal.
